<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hasil_detail</class>
 <widget class="QMainWindow" name="hasil_detail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hasil Sniffing</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #90caf9, stop:1 #a6d4fa);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Lato Heavy</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Kuis</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>30</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Jawab pertanyaan dibawah dengan memilih salah satu pilihan yang ada kemudian menekan tombol lanjut. </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>30</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>30</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="soal_kuis">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Soal</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="pilihan1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Pilihan 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="pilihan2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Pilihan 2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="pilihan3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Pilihan 3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="pilihan4">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Pilihan 4</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="0">
           <widget class="QPushButton" name="kembaliBtn">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">padding: 4px 15px;
font-size: 11px;
text-align: center;
cursor: pointer;
outline: none;
color: #fff;
background-color: #3949AB;
border: none;
border-radius: 15px;
box-shadow: 0 9px #999;</string>
            </property>
            <property name="text">
             <string>Menu Awal</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>resource/1x/Asset 2.png</normaloff>resource/1x/Asset 2.png</iconset>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QPushButton" name="nilaiBtn">
            <property name="styleSheet">
             <string notr="true">padding: 4px 15px;
font-size: 11px;
text-align: center;
cursor: pointer;
outline: none;
color: #fff;
background-color: #3949AB;
border: none;
border-radius: 15px;
box-shadow: 0 9px #999;</string>
            </property>
            <property name="text">
             <string>Cek Nilai</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButton_2">
            <property name="styleSheet">
             <string notr="true">padding: 4px 15px;
font-size: 11px;
text-align: center;
cursor: pointer;
outline: none;
color: #fff;
background-color: #306ea8;
border: none;
border-radius: 15px;
box-shadow: 0 9px #999;</string>
            </property>
            <property name="text">
             <string>Selanjutnya</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="sebelumBtn">
            <property name="styleSheet">
             <string notr="true">padding: 4px 15px;
color: rgb(37, 179, 255);
font-size: 11px;
text-align: center;
cursor: pointer;
outline: none;
color: #fff;
background-color: #306ea8;
border: none;
border-radius: 15px;
box-shadow: 0 9px #999;</string>
            </property>
            <property name="text">
             <string>Sebelum</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>100</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
